engine/installation/linux/oracle.md

```diff
@@ -203,9 +229,10 @@ users to run Docker commands and for other optional configuration steps.
 
 #### Upgrade Docker
 
-To upgrade Docker, first run `sudo yum makecache fast`, then follow the
-[installation instructions](#install-docker), choosing the new version you want
-to install.
+To upgrade Docker, run `sudo yum update docker-engine` to upgrade to the latest available
+version. If you would prefer to upgrade to a specific version, first run 
```

Comment:
Yes, I was imagining a construction like:

To upgrade Docker to the latest version, run `sudo yum update docker-engine`. To upgrade to a specific version, first run....